pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: zds_flutter
 description: Zebra Design System components for Flutter. A selection of UI components from Zebra Technologies.
-version: 1.3.1
+version: 2.0.0
```

Comment:
I have set up [release-please](https://github.com/googleapis/release-please) to manage our version publishing - rather than change this here we should use the release please prs to bump the major version. I can do that once this is merged

Reply:
reverted back to 1.3.1